src/workers/filesystems/remote-filesystem.ts

```diff
@@ -218,15 +221,22 @@ export function getRemoteFilesystem({
     async deleteFile(name: string): Promise<void> {
       const fileInCache = cache[name];
 
-      try {
-        await httpRequest(
-          `${process.env.API_URL}/api/storage/folder/${fileInCache.parentId}/file/${fileInCache.id}`,
-          { method: 'DELETE', headers }
-        );
-      } catch (err) {
+      const result = await clients.newDrive.post(
+        `${process.env.NEW_DRIVE_URL}/drive/storage/trash/add`,
+        {
+          items: [
+            {
+              type: 'file',
+              id: fileInCache.fileId,
```

Comment:
Does the trash require the `fileId` from the network instead of the `file.id`?

Reply:
Yes